.github/workflows/test-warehouse.yml

```diff
@@ -78,6 +78,7 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: "3.8.17"
+          cache: "pip"
```

Comment:
Cool
Did you see performance improvements in the job?
I tried adding it once to another workflow and it didn't save as much as I hoped, but still I guess it's a good configuration